defaults/defaults.go

```diff
@@ -129,7 +129,7 @@ var MOBILEUSERAGENTS = []string{
 	"Mozilla/5.0 (iPhone; CPU iPhone OS 8_4 like Mac OS X) AppleWebKit/600.1.4 (KHTML, like Gecko) Version/8.0 Mobile/12H143 Safari/600.1.4",
 }
 
-var DEFAULTPAGEVIEWFIELD = "urihash"
+var DEFAULTPAGEVIEWFIELD = "permanentid"
```

Comment:
why change the default field to `permanentid`?

Reply:
It is not the recommended field by ML: 

https://onlinehelp.coveo.com/en/cloud/about_the_permanentid_field.htm

Reply:
That permanentid is a very new field, only exists on new sources and only on cloudv2.
Keep urihash, and set it as the field yourself if you want to.

This is only important if you are actually sending a permanentid yourself to your UA data and don't want duplicate data. And if you are in the stage of having ua data already for ML, you can probably stop the bot?

Reply:
That permanentid is a very new field, only exists on new sources and only on cloudv2.
Keep urihash, and set it as the field yourself if you want to.

This is only important if you are actually sending a permanentid yourself to your UA data and don't want duplicate data. And if you are in the stage of having ua data already for ML, you can probably stop the bot?

Reply:
I'm fine with changing it back, but the documentation says 

> All Coveo Cloud V2 standard sources automatically include the permanentid metadata and field mapping after the May 5, 2017 release.

So a year-old change is still too much "very new" to be put as default? 😛

Reply:
Most of my scenarios still push on cloudV1, so yes.

As I said, the only change with this permanentid vs urihash was because from time to time if the docs would change uri, they could change urihash which meant ML couldn't find them again.

Urihash was not necessarily unique either. Since you could index twice the same thing in the same org.

But whichever param you use for this is not relevant, all you need is a field allowing you to uniquely identify results for ML. And as I said for the bot, if you start having actual data with permanentId you probably should stop using the bot and use real data.

So to stay retro-compatible with cloudV1, we keep urihash.